sos/plugins/kernel.py

```diff
@@ -81,7 +82,10 @@ def setup(self):
             "/proc/timer*",
             "/proc/lock*",
             "/proc/misc",
-            "/var/log/dmesg"
+            "/var/log/dmesg",
+            numa_path + "node*/meminfo",
+            numa_path + "node*/cpulist",
+            numa_path + "node*/distance"
```

Comment:
Why would we put all this in the `kernel` plugin, when everything else numa-related is in `numa`? It seems unnecessarily confusing...

To put this into the `numa` plugin all you'd need to do would be to add a file trigger for a relevant path.

Generally we try to keep plugins small and specific today - `kernel` is one that has been around a very long time and is a "catch all" with no trigger conditions and a pretty weak organisation so we try not to add stuff there unless there is really nowhere else for it to go.

(I'd be very open, for e.g., to a patch moving all the remaining memory `/proc` files to `memory`, and splitting off driver, interrupt, and other bits to their own, dedicated plugins).